src/Tribe/Rewrite.php

```diff
@@ -68,28 +68,31 @@ class Tribe__Rewrite {
 	 * An array cache of resolved canonical URLs in the shape `[ <url> => <canonical_url> ]`.
 	 *
 	 * @since 4.9.11
+	 * @since TBD    Changed default value to an empty array to prevent potential fatals.
 	 *
 	 * @var array
 	 */
-	protected $canonical_url_cache = null;
+	protected $canonical_url_cache = [];
```

Comment:
We could also type hint them
```suggestion
	protected array $canonical_url_cache = [];
```